app/View/Components/faq-item--ui.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\View\Components;
+
+use Closure;
+use Illuminate\Contracts\View\View;
+use Illuminate\View\Component;
+
+class faq-item--ui extends Component
```

Comment:
Should be `FaqItemUI` or something like that. When generating components using artisan, Use the PascalCase format to name it. Artisan will name your view in lower kebab case. Experiment with it for a while. You can easily delete the generated classes and views without any consequences.

Reply:
Avoid the double minus sign `--` in the name